src/Config.php

```diff
@@ -937,7 +937,10 @@ public static function getLanguage($lang)
      **/
     public function showFormFieldUnicity()
     {
-
+        Toolbox::deprecated(
+            "This method will be removed in the next version",
+            version: "11.1.0"
+        );
```

Comment:
```suggestion
        Toolbox::deprecated(
            message: "This method will be removed in the next version",
            version: "11.1.0"
        );
```
If using named parameters, it is better to use them for all of the parameters.